pkg/products/grafana/prometheusRules.go

```diff
@@ -2,44 +2,37 @@ package grafana
 
 import (
 	"fmt"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
 	"github.com/integr8ly/integreatly-operator/pkg/resources"
 	l "github.com/integr8ly/integreatly-operator/pkg/resources/logger"
 	monv1 "github.com/rhobs/obo-prometheus-operator/pkg/apis/monitoring/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, namespace string) resources.AlertReconciler {
-	installationName := resources.InstallationNames[installType]
+func GrafanaAlertReconciler(logger l.Logger, installation *integreatlyv1alpha1.RHMI) resources.AlertReconciler {
```

Comment:
Why are we changing this and moving to bootstrap?
Can we not follow same flow as it is for example in sso - https://github.com/integr8ly/integreatly-operator/blob/0b37f6b3c92ca7a0b85389415c5e9568446efcf7/pkg/products/rhsso/reconciler.go#L235 ?

Reply:
done. Thank you